test/Transformations_test.jl

```diff
@@ -154,13 +152,32 @@
 
         # Test that the resulting tn contains no tensors with larger rank than the original
         rank = length ∘ size ∘ parent
-        @test max(rank(tensors(reduced)) == max(rank(tensors(tn))))
+        @test max(rank.(tensors(reduced))...) <= max(rank.(tensors(tn))...)
```

Comment:
Would you mind deleting `rank` and replacing it for `ndims`?

Reply:
Also it's better if you call `maximum`: the same as `max` but without variadic arguments and can use some `map`.
```suggestion
        @test maximum(ndims, tensors(reduced)) <= maximum(ndims, tensors(tn))
```